### PR TITLE
Issue 32595 - Kafka Streams Dev UI migration to v2

### LINKED ISSUE #32595: Dev UI: Migrate Kafka streams to the new Dev UI
### Description

From https://github.com/quarkusio/quarkus/issues/31315

### Implementation ideas

_No response_

### PR DESCRIPTION
**Pull request by dcotfr**
Proposal of rewriting of Kafka Streams Dev UI for v2.

Replaces Mermaid usage by a wasm version of Graphviz (seems lighter and with less dependencies).

Still generates a mermaid graph definition (for backward compatibility) and adds a new digraph definition for Graphviz.

Warning : contains a direct link to https://cdn.jsdelivr.net/npm/@hpcc-js/wasm/dist/graphviz.js in qwc-kafka-streams-topology.js, because I don't know how to include this dependency with in a more cleaner / secured way.

fixes #32595

### CODE REVIEW

**Comment by gsmet** [comment]
Thanks a lot, I pinged @phillip-kruger so that he can have a look.

File: extensions/kafka-streams/deployment/src/main/resources/dev-ui/qwc-kafka-streams-topology.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
+import { QwcHotReloadElement, html, css } from 'qwc-hot-reload-element';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { JsonRpc } from 'jsonrpc';
+
+import { Graphviz } from "https://cdn.jsdelivr.net/npm/@hpcc-js/wasm/dist/graphviz.js";
```
Comment (phillip-kruger) [inline extensions/kafka-streams/deployment/src/main/resources/dev-ui/qwc-kafka-streams-topology.js:5]: Here you can use `import { Graphviz } from "@hpcc-js/wasm/graphviz";` and add this to deployment/pom.xml:

```
<dependency>
        <groupId>org.mvnpm.at.hpcc-js</groupId>
        <artifactId>wasm</artifactId>
        <version>2.14.1</version>
        <scope>runtime</scope>
      </dependency>
```

You can also add above to build parent pom dependency management and reference in the extension pom without the version

**Review by phillip-kruger — CHANGES_REQUESTED** [review]
LGTM, I think we can merge once the change is made

**Review by phillip-kruger — APPROVED** [review]

**Comment by phillip-kruger** [comment]
@dcotfr Thanks for this ! Please rebase and squash your PR, and then on green CI we can merge

**Comment by phillip-kruger** [comment]
@dcotfr seems like the wasm js dependency pulls in a lot of other dependencies with ranges. To make our build faster we lock these in the build-parent. I'll send you (a bit later) what needs to be added to the build parent.

**Pull request closed without merging**